config-linux.md

```diff
@@ -728,7 +728,7 @@ The values MUST be absolute paths in the [container namespace][container-namespa
 [cgroup-v1-net-prio]: https://www.kernel.org/doc/Documentation/cgroup-v1/net_prio.txt
 [cgroup-v1-pids]: https://www.kernel.org/doc/Documentation/cgroup-v1/pids.txt
 [cgroup-v2]: https://www.kernel.org/doc/Documentation/cgroup-v2.txt
-[devices]: https://www.kernel.org/doc/Documentation/devices.txt
+[devices]: https://www.kernel.org/doc/Documentation/cgroup-v1/devices.txt
```

Comment:
This link (defining major and minor numbers) is for the device listing, which is [here][1] since [07c7e30c (Documentation/admin-guide: split the device list to a separate file, 2016-11-03)][2].

[1]: https://www.kernel.org/doc/Documentation/admin-guide/devices.txt
[2]: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=07c7e30c1885393b07efcaf62d51b219755b6bf5